### ISSUE TITLE
Make the output format more obvious to find

### ISSUE THREAD

**Issue by iMartyn** [segmentio/terraform-docs#180]
### Prerequisites

Put an `x` into the box that applies:

- [ ] This issue describes a bug.
- [ ] This issue describes a feature request.
- [x] This issue describes a documentation improvement request.

For more information, see the [Contributing Guidelines](https://github.com/segmentio/terraform-docs/tree/master/CONTRIBUTING.md).

### Description

It took me a while to find the place in the repo to find the output format.  I've discovered this repo because someone at work was rejoicing that it supports tf-0.12.x now, and so I came to look and see what it is.

#### Actual Behavior

Confusion, lack of desire, uncertainty

#### Expected Behavior

Excitement, desire, feeling of "must have".

At least examples from the docs can be linked to from the `Readme.md`.  It would be nice if the markdown examples were previewable in github so you get a feel for what the rendered output looks like too.

### Replication Case

Be someone who doesn't know what this tool is ;-)

### Version
`master`

**Comment by aaronsteers** [comment]
@iMartyn - I just stumbled on this tool yesterday and had the same frustration so I opened PR #216 which has a fix.

Even after I located the correct docs, I consistently had trouble returning to that page. A more robust job could be done (possibly remove USAGE_GUIDE.md or migrate content from `docs/README.md`), but I've at least proposed a minimal fix so users would always get to the right place.
